app/components/Modal/Modal.css

```diff
@@ -22,28 +13,37 @@ html[data-theme='dark'] .backdrop {
 }
 
 .content {
+  line-height: 1.3;
   position: fixed;
-  max-width: 680px;
-  top: 20px;
-  left: 0;
-  right: 0;
-  margin: 0 auto;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 425px;
```

Comment:
is this max-width an effective width? If not will content that does not fit in this clip the modal box?

Reply:
Yes, but you can override it. The photo gallery view modal does this. It's just that it needs a specified width, and might as well set a max as good practice or something.

Reply:
It doesn't make more sense to have it be a `min-width` then? If it should have a "default" but allow to grow according to the content?

Reply:
Not really, since the contents of the modals often use `width: 100%` and that'll look for the maximum width, not the minimum.